src/pages/en/guides/cms/storyblok.md

```diff
@@ -379,20 +390,24 @@ export async function getStaticPaths() {
         slug: story.full_slug === 'home' ? undefined : story.full_slug
       },
       props: {
-        content: story.content
+        story
       }
     }
   })
   return pages
 }
-const { content } = Astro.props
+
+const { story } = Astro.props
 ---
 <html lang="en">
   <head>
     <title>Storyblok & Astro</title>
   </head>
   <body>
-    <StoryblokComponent blok={content} />
+    {story.full_slug === 'home' && (
```

Comment:
For the reader's benefit, I think we should keep the slug page as simple as possible. In this case, what do you think about removing the h1 element but keeping the story to be passed as props? Then the reader can use all the metadata stored in the story to build their page. 🎉

Reply:
When I was building this tutorial, and debugging some of the issues above, I was struggling to see the difference between the homepage and the blog posts

As they both use the same file, it wasn't clear which I was looking at at any given time, or which was causing errors.

This isn't a problem with storyblok or astro really, the usage of `[...slug]` is correct, but I think that being a little redundant here can actually help people getting started if they have any problems.

If we want to handhold their mistakes, I think the title is helpful, but if we don't then we could let them figure out this debugging step themselves.